pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -612,7 +612,6 @@ https://github.com/echasnovski/mini.starter/,HEAD,
 https://github.com/echasnovski/mini.statusline/,HEAD,
 https://github.com/echasnovski/mini.surround/,HEAD,
 https://github.com/echasnovski/mini.tabline/,HEAD,
-https://github.com/echasnovski/mini.test/,HEAD,
```

Comment:
Why?

Reply:
Because it isn't supposed to be there. Was removed in https://github.com/NixOS/nixpkgs/pull/387101